proto/sge/strategicreserve/tx.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package sgenetwork.sge.strategicreserve;
+
+import "sge/strategicreserve/feegrant.proto";
+
+option go_package = "github.com/sge-network/sge/x/strategicreserve/types";
+
+// Msg defines the Msg service.
+service Msg {
+  // InvokeFeeGrant defines a method to add a feegrant with the given data.
+  rpc InvokeFeeGrant(MsgInvokeFeeGrant) returns (MsgInvokeFeeGrantResponse);
+  // RevokeFeeGrant defines a method to add a feegrant with the given data.
+  // rpc RevokeFeeGrant(MsgRevokeFeeGrant) returns (MsgRevokeFeeGrantResponse);
```

Comment:
this rpc is commented out. It would not work

Reply:
Feature removed #153